raw-spans-grouper/raw-spans-grouper/src/main/java/org/hypertrace/core/rawspansgrouper/RawSpansGrouper.java

```diff
@@ -86,6 +89,13 @@ StreamsBuilder buildTopologyWithClock(
                 Stores.persistentKeyValueStore(SPAN_STATE_STORE_NAME), keySerde, valueSerde)
             .withCachingEnabled();
 
+    StoreBuilder<KeyValueStore<IpIdentity, SpanMetadata>> ipsToSpanMetadataStateStoreBuilder =
+        Stores.keyValueStoreBuilder(
```

Comment:
What if we keep in memory store only? say a cache. what would be pros and cons.?

Reply:
discussed pros and cons separately. going with SS for now